sdks/python/container/Dockerfile

```diff
@@ -79,14 +80,18 @@ RUN \
     pip install "tensorflow == 1.4.0" && \
     pip install "protorpc == 0.11.1" && \
     pip install "python-gflags == 3.0.6" && \
-    # Remove pip cache.
```

Comment:
Small comment: deletes have to be in the same RUN as the installs to avoid the removed files to take up space in the image. Each RUN creates an additional layer with changes.

Reply:
@holdenk please address this comment.